users/tracing/users.go

```diff
@@ -44,6 +44,14 @@ func (urm userRepositoryMiddleware) Save(ctx context.Context, user users.User) e
 	return urm.repo.Save(ctx, user)
 }
 
+func (urm userRepositoryMiddleware) Update(ctx context.Context, user users.User) error {
+	span := createSpan(ctx, urm.tracer, saveOp)
+	defer span.Finish()
+	ctx = opentracing.ContextWithSpan(ctx, span)
+
+	return urm.repo.Update(ctx, user)
```

Comment:
Does this update whole user structure or just Metadata?

If it updates whole user structure, it means that User can change his own "constructive" information - which is contrary to the ide that users are created and managed by the admins - and can lead to security breaches.

@nmarcetic please review this one and advise.

Reply:
@nmarcetic please review and advise here so we can move further.